### ISSUE TITLE
Add Stonyhurst coordinate system

### ISSUE THREAD

**Issue by jgieseler** [jgieseler/solarmach#1]
Stonyhurst coordinate system is right now only included in the streamlit part (cf. https://github.com/jgieseler/Solar-MACH/issues/6). Move this functionality into the **solarmach** package (and adjust the streamlit version accordingly).

**Comment by jgieseler** [comment]
How it's done at the moment in the streamlit version:
```python
# if input coordinates for reference are Stonyhurst, convert them to Carrington for further use
import astropy.units as u
from astropy.coordinates import SkyCoord
from sunpy.coordinates import frames
date = '2021-10-28 15:15:00'
reference_long = 2                               # Stonyhurst longitude of reference (None to omit)
reference_lat = 26                               # Stonyhurst latitude of reference (None to omit)
coord = SkyCoord(reference_long*u.deg, reference_lat*u.deg, frame=frames.HeliographicStonyhurst, obstime=date)
coord = coord.transform_to(frames.HeliographicCarrington(observer='Sun'))
reference_long = coord.lon.value                 # Carrington longitude of reference
reference_lat = coord.lat.value                  # Carrington latitude of reference
```

**Comment by jgieseler** [comment]
I think in the long run, we should aim at completely replacing the `show_earth_centered_coord` parameter of `SolarMACH.plot()` with a new `coord_sys='Carrington'` parameter for `SolarMACH()` (that by default is 'Carrington').

So an example should look like this:

``` python
from solarmach import SolarMACH

# necessary
body_list = ['STEREO-A', 'Earth', 'BepiColombo', 'PSP', 'Solar Orbiter', 'Mars']
vsw_list = [400, 400, 400, 400, 400, 400, 400]
date = '2021-10-28 15:15:00'

# optional
cood_sys = 'Stonyhurst'  # by default 'Carrington'

# initialize
sm = SolarMACH(date, body_list, vsw_list, cood_sys)

# make plot
sm.plot()
```

**Comment by jgieseler** [comment]
Closed with [Release 0.1.3](https://github.com/jgieseler/solarmach/releases/tag/v0.1.3)
